lib/board_transformer.rb

```diff
@@ -0,0 +1,34 @@
+module BoardTransformer
+    def horizontal_rows(board)
+        return board.positions.each_slice(board.size).to_a
+    end
+    
+    def vertical_rows(board)
+        max_positions = board.max_positions
+        size = board.size 
+        (0..size - 1).map do |starting_row|
```

Comment:
This can be `(0...size)`. Representing a range with two dots will include the last value and omit it if you use three dots.